corehq/apps/users/templates/users/extra_users_confirm_billing.html

```diff
@@ -3,9 +3,7 @@
 {% load hq_shared_tags %}
 {% load crispy_forms_tags %}
 
-{% block js %}{{ block.super }}
-  <script src="{% static 'accounting/js/widgets.js' %}"></script>
-{% endblock %}
+{% requirejs_main 'accounting/js/widgets' %}
```

Comment:
Just want to double check that this is ok. This html page inherits from `users/base_template.html`. That page can also use `requirejs_main` instead, but does it present an issue to use `requirejs_main` in a child class of a parent who already uses it?

Reply:
Does [this page](https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/migrating.md) answer your question?

Reply:
Somewhat. This line somewhat answers the question:
> Most often, there's already a single script that's only included on the page you're migrating, which you can use as the main module.

But leads me to believe that if inheriting from a base template that already defines a main module, you cannot define another main module in the subclass.

Reply:
Alright based on 
> HTML files that are only used as the base for other templates don't need to have a main module or a requirejs_main tag.

I [reverted my change](36f8b6b011d290ca09c79b09f967fd0fc48567ac) to the base_template file.

Reply:
Reverting the change to the base template seems right, sorry I didn't notice that before.

I'm wondering if this is using `requirejs_main` correctly. That template tag is intended to load the main entrypoint script for the page. In this case `accounting/js/widgets` doesn't look like a main entrypoint script. I'm not familiar enough with the accounting code to know, but maybe you want something like [base_subscriptions_main.js](https://github.com/dimagi/commcare-hq/blob/36f8b6b011d290ca09c79b09f967fd0fc48567ac/corehq/apps/accounting/static/accounting/js/base_subscriptions_main.js)?

Reply:
Sorry for missing this. I skimmed through and assumed the question related to the previous use of `requirejs_main`, not the current use. Like you said on slack, other pages include accounting widgets the same way. My understanding is as long as a JS file uses hqDefine, it can be included as a main entry point.

Reply:
> My understanding is as long as a JS file uses hqDefine, it can be included as a main entry point.

Confirming this is true - please do reword the docs if they could be clearer. (There are also [these](https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/dependencies.md) newer docs which are less migration-oriented.)